### DISCUSSION TITLE
Release Cycle Plan

### DISCUSSION THREAD

**Discussion by alokprasad7** [DMTF/libspdm#1172] [General]
Is there a plan to have a new tag release ( 2.1.1) in the coming days?
How you guys decide for a new tag release ? Fix date or timeline?

**Comment by jyao1** [comment]
The plan is to do quarterly release. (increase minor version or major version)
Tag is created at the end of each quarter by default, such as March 31, June 30, September 30, December 31.

If you have special request for patch release, please attend the regular SPDM Code TF meeting to discuss.
